mock/py/mockbuild/plugins/lvm_root.py

```diff
@@ -382,6 +382,21 @@ def hook_remove_snapshot(self, name):
         lvm_do(['lvremove', '-f', self.vg_name + '/' + lv_name])
         self.buildroot.root_log.info("deleted {name} snapshot".format(name=name))
 
+    def hook_postupdate(self):
+        name = self.get_current_snapshot()
+        if re.match(r".*_updated_[\-\_0-9]{16}$", name):
+            name = re.sub(r"_updated_[\-\_0-9]{16}$", "", name)
+
+        timestamp = time.localtime(time.time())
+        name = name + "_updated_{y}-{mo:02d}-{d:02d}_{h:02d}-{m:02d}".format(
+            y=timestamp.tm_year, mo=timestamp.tm_mon, d=timestamp.tm_mday,
+            h=timestamp.tm_hour, m=timestamp.tm_min
+        )
+
+        self.make_snapshot(name)
```

Comment:
We probably need to call set_current_snapshot?

Reply:
`set_current_snapshot` is called in `make_snapshot`.

Reply:
Ah, I was mistaken by `hook_postinit` which (mistakenly) re-calls the `set_current_snapshot` method.

Reply:
Can you please quickly review #596?